src/tools/handlers.py

```diff
@@ -121,6 +140,8 @@ def add_address(args, book: AddressBook):
     record.add_postal_address(address)
     return f"{name}`s address is added"
 
+
+@input_error
```

Comment:
Here should be other decorator.
Currently, if you enter only name, you'll get error "Please give me name and phone"
For address field other decorator should be implemented like for the not.